Governance/concepts/license-qa.md

```diff
@@ -1,15 +1,15 @@
 # Q&A
 * <ins>Does a license last forever?</ins><br>
-Technically a license is valid until you hold the copyright, which usually in Australia expires 70 years after creation. You can revoke a license and apply a different one anytime<br> 
+Technically a license is valid while(?) you hold the copyright, which usually in Australia expires 70 years after creation. You can revoke a license and apply a different one anytime<br> 
```

Comment:
I wrote this originally and using "while" is correct in this context, if you want to double check the information in general then it might be useful having Katie's comment on this.

The other question mark, again is something I wrote and using "collaborators" should work, it was just an example of the kind of "id" you can refer to, and I guess I was thinking of collaborators through their researcher-id and ended up writing researcher instead

Reply:
Licenses don't last forever. I'm guessing that an organisation could change the terms of the license that was issued, but I'll need to check this. In Australia, copyright lasts for 70 years after the death of the creator unless the work was created before 2005 when it's 50 years. https://www.nla.gov.au/using-library/copyright-library-collections/how-long-does-copyright-last.

Reply:
So I can change the "70 years after death (50 years if license applied before 2005" while "organisation could change the terms of the license that was issued" is expressed in:
"You can revoke a license and apply a different one anytime" I'm quite certain of this, but it can't hurt to double check

Reply:
Done, I also notice the second question clarifies what happens if a license is changed